js/api/Triggers.js

```diff
@@ -32,32 +34,80 @@ export function loadTriggers(form, survey, callback) {
   })
 }
 
-// Fetches the cached forms related to a specific survey
-export function loadCachedTrigger(formId) {
-  return realm.objects('TimeTrigger').filtered(`formId= "${formId}"`)
+// Fetches the cached triggers related to a specific survey.
+export function loadCachedTriggers(formId) {
+  return realm.objects('TimeTrigger').filtered(`formId= "${formId}"`);
+}
+
+/**
+ * Fetches all cached datetime triggers for the accepted surveys
+ * @return {object}  Realm object containing an array of 'TimeTrigger' objects,
+ */
+export function loadCachedTimeTriggers(options = {}, callback) {
+  loadAllAcceptedSurveys((err, response) => {
+    if (err) {
+      console.warn(err);
+      callback(err, []);
+      return
+    }
+    
+    try {
```

Comment:
I believe this is the same try/catch block that we had fixed yesterday. This branch needs a quick update with `master-v2`